pages/services/babydegen.jsx

```diff
@@ -92,25 +107,28 @@ const WhatIsBabyDegen = () => (
 );
 
 const MeetTheOptimusAgent = () => (
-  <SectionWrapper customClasses="border-y border-b-0">
+  <SectionWrapper
+    customClasses="border-y border-b-0 border-[#FFE0E6]"
+    backgroundType="LINEAR_GRADIENT"
```

Comment:
I think, there will be only one place where we'll have the red gradient, can we just use those class names in `customClasses`, instead of having it as a separate `backgroundType`?

Reply:
This makes sense! I've changed changed that in the latest push now :)